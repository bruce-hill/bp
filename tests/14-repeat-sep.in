foo(1)
foo(1,2,3,4)
foo(23,34,56)
foo(xxx)
foo(,,1,,)
xxx
,,,
1,2,3
