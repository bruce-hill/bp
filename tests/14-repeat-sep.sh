# The '%' operator modifies repeating patterns, allowing you to give a separator between matches
# Example: bp -p '+"x" % ":"' will match "x", "x:x", "x:x:x", etc.
bp -p '`( +int % `, `)'
