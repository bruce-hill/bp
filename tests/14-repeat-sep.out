foo(1)
foo(1,2,3,4)
foo(23,34,56)
